### ISSUE TITLE
networkx 2 compatibility

### ISSUE THREAD

**Issue by yoavram** [elfi-dev/elfi#326]
elfi is incompatible with networkx 2, but networkx 1.11 is incompatible with matplotlib 3.

This means that it is impossible to plot with networkx and matplotlib 3 if using Elfi in the same environment.

To reproduce, install elfi, matplotlib 3.
```python
import networkx as nx
G = nx.random_regular_graph(2, 100)
nx.draw(G)
```
this leads to:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-29-34f682212305> in <module>
      1 G = nx.random_regular_graph(2, 100)
----> 2 nx.draw(G)

~/miniconda3/envs/ENV/lib/python3.7/site-packages/networkx/drawing/nx_pylab.py in draw(G, pos, ax, hold, **kwds)
    124     if 'with_labels' not in kwds:
    125         kwds['with_labels'] = 'labels' in kwds
--> 126     b = plt.ishold()
    127     # allow callers to override the hold state by passing hold=True|False
    128     h = kwds.pop('hold', None)

**AttributeError: module 'matplotlib.pyplot' has no attribute 'ishold'**
```

ELFI Version: 0.7.5
matplotlib version: 3.2.1
networkx version: 1.11
Python Version: 3.7.6
OS: MacOS

**Comment by danielward27** [comment]
I have also have had this compatibility issue arise with pomegranate.

**Comment by hpesonen** [comment]
Hi! We have now migrated from networkx 1.X to networkx 2.X on dev-branch, and we'll update master soon. The compatibility issue should now have been resolved.

**Comment by wleoncio** [comment]
Issue potentially fixed on 6fae9719609b73fc0ea89db62ccea1ef5d1b16f5, please wait for a new release (or try the `dev` channel) and reopen this issue if the problem persists.
